src/test/java/dev/ocpd/jsensible/arch/JpaRulesArchTest.kt

```diff
@@ -0,0 +1,20 @@
+package dev.ocpd.jsensible.arch
```

Comment:
Incomplete test cases. It should at least two cases. eg: 'using spring's nullable' and 'using jetbrain's nullable'